apps/flow-docs/app/routes/api/flips.tsx

```diff
@@ -0,0 +1,77 @@
+import type { LoaderFunction } from "@remix-run/node";
+import { useLoaderData } from "@remix-run/react";
+import type { FlipCellProps } from "../../../../../libs/design-system/dist/lib/Components/Flips/FlipCell";
+import { Octokit } from "@octokit/core";
+
+export type LoaderData = FlipCellProps[];
+
+export const loader: LoaderFunction = async () => {
+  const octokit = new Octokit({
+    auth: process.env.PERSONAL_ACCESS_TOKEN,
```

Comment:
Be sure to rename this env var  to: `BOT_GITHUB_TOKEN` to work with the deployment.